### DISCUSSION TITLE
Reducing logging verbosity

### DISCUSSION THREAD

**Discussion by JonTheBoy** [blakeblackshear/frigate#8728] [Ask A Question]
I have altered the logging configuration for Frigate NVR in an attempt to reduce verbosity:
```
logger:
  default: warning
  logs:
    frigate.object_detection: info
```

I am trying to remove the repeated web server access logs like these:

> 2023-11-23 04:48:00.250166115  1.1.1.1 - - [23/Nov/2023:04:48:00 +0000] "GET /api/stats HTTP/1.1" 200 2603 "-" "HomeAssistant/2023.11.3 aiohttp/3.8.5 Python/3.11" "-"
2023-11-23 04:48:08.253958584  1.1.1.1 - - [23/Nov/2023:04:48:08 +0000] "GET /api/stats HTTP/1.1" 200 2671 "-" "HomeAssistant/2023.11.3 aiohttp/3.8.5 Python/3.11" "-"
2023-11-23 04:48:16.276499743  1.1.1.1 - - [23/Nov/2023:04:48:16 +0000] "GET /api/stats HTTP/1.1" 200 2602 "-" "HomeAssistant/2023.11.3 aiohttp/3.8.5 Python/3.11" "-"

I see no value in logging every web call here. And filling the logs with lots of noise makes it more difficult to find things that are of interest. How do I turn the logging for this component to WARNING or ERROR? (I'm guessing the entries are being logged by nginx but given I've set default to warn, I'm surprised to be seeing these entries still.)

It would be nice if the log entries included level (e.g. error, warn, info) and source, so I might need to put that into suggestions.

**Comment by NickM-27** [comment]
nginx logs in general are helpful in a number of circumstances. In frigate 0.13 the stats and frigate version calls are not logged in the nginx logs

**Reply by JonTheBoy** [reply]
I get that nginx logs can be helpful in some circumstances. I'd just like to be able to adjust logging to suit my circumstances, like I can for other components. Good to know that 0.13 stops logging stats and version calls.
In my case, all calls to the Frigate Docker instance will be fronted by another nginx instance which acts as a reverse proxy (RP). No doubt I will find some other challenges with doing so but this approach suits my usage. And logging in this RP nginx instance is what is useful for me.
